### ISSUE TITLE
Better controls visibility

### ISSUE THREAD

**Issue by mehulmpt** [vime-js/vime#24]
Controls are not much visible if the video running is white. See the screenshot for more details:

<img width="285" alt="Screenshot 2020-04-21 at 12 54 31 AM" src="https://user-images.githubusercontent.com/10388889/79791272-aa96c100-836a-11ea-9dd9-b4467594968e.png">

Maybe a `1px` black shadow or outline could be added to the controls?

**Comment by mihar-22** [comment]
What version are you using @mehulmpt? In the latest (1.5.0) I made some changes to make them more visible. Is it still not good enough?

**Comment by mehulmpt** [comment]
Yeah, I was using `1.4.4` at the moment. I've updated to `1.5.0` and it looks better, thanks!
